server/resolvers/TestPlanReport/finalizedTestResultsResolver.js

```diff
@@ -8,6 +8,9 @@ const finalizedTestResultsResolver = async (testPlanReport, _, context) => {
   // Return the primary test plan run, otherwise pick the first TestPlanRun found.
   const testPlanRun =
     testPlanReport.testPlanRuns.find(({ isPrimary }) => isPrimary) ||
+    testPlanReport.testPlanRuns.find(testPlanRun =>
+      testPlanRun.testResults?.some(testResult => !!testResult.completedAt)
+    ) ||
```

Comment:
For reviewers: Found odd behaviors when doing the following:
* Assigning self to report row on Test Queue
* Assigning other tester to report row on Test Queue
* Opening the report as the other tester navigating and this resolver could silently throw errors when navigating so attempted to resolve that